src/shared/src/types/change-request-types.ts

```diff
@@ -112,8 +112,8 @@ export interface ProposedSolutionCreateArgs {
 export interface WBSProposedChangesCreateArgs {
```

Comment:
go ahead and delete this/ incorporate the necessary fields into ProjectProposedChangesCreateArgs